src/iotjs_binding.c

```diff
@@ -446,17 +447,16 @@ iotjs_jval_t iotjs_jhelper_eval(const char* name, size_t name_len,
 
 
 #ifdef ENABLE_SNAPSHOT
-iotjs_jval_t iotjs_jhelper_exec_snapshot(const void* snapshot_p,
-                                         size_t snapshot_size, bool* throws) {
-  jerry_value_t res = jerry_exec_snapshot(snapshot_p, snapshot_size, false);
+iotjs_jval_t iotjs_exec_snapshot(uint32_t snapshot_function_idx, bool* throws) {
+  /* iotjs_js_modules_{s,l} is generated by the js2c.py */
+  jerry_value_t result =
+      jerry_exec_snapshot_at((const void*)iotjs_js_modules_s,
+                             iotjs_js_modules_l, snapshot_function_idx, false);
   /* the snapshot buffer can be referenced
    * until jerry_cleanup is not called */
-
-  *throws = jerry_value_has_error_flag(res);
-
-  jerry_value_clear_error_flag(&res);
-
-  return iotjs_jval_create_raw(res);
+  *throws = jerry_value_has_error_flag(result);
+  jerry_value_clear_error_flag(&result);
+  return iotjs_jval_create_raw(result);
```

Comment:
I guess that `iotjs_jval_create_raw` is planned to be removed. Can we simply do `return result`?

Reply:
Yes, please fix it.